services/convert/repository.go

```diff
@@ -181,6 +181,7 @@ func innerToRepo(ctx context.Context, repo *repo_model.Repository, permissionInR
 		Parent:                        parent,
 		Mirror:                        repo.IsMirror,
 		HTMLURL:                       repo.HTMLURL(),
+		URL:                           repo.URL(),
```

Comment:
What's the difference between calling `repo.HTMLURL()` and `repo.URL()`?

Reply:
there is no difference. I made a new function for it if the repo.HTMLURL() function changed latter. remove it or keep?

Reply:
Just reuse `repo.HTMLURL`

Reply:
thanks for the suggestion, reused that.